src/ansys/geometry/core/tools/repair_tools.py

```diff
@@ -177,6 +181,46 @@ def find_inexact_edges(self, bodies: List["Body"]) -> List[InexactEdgeProblemAre
             for res in problem_areas_response.result
         ]
 
+    def find_short_edges(
+            self, bodies: List["Body"], length: Real = 0.0
+    ) -> List[ShortEdgeProblemAreas]:
+        """Find the short edge problem areas.
+
+        This method finds the short edge problem areas and returns a list of
+        short edge problem areas objects.
+
+        Parameters
+        ----------
+        bodies : List[Body]
+            List of bodies that short edge are investigated on.
+
+        Returns
+        -------
+        List[ShortEdgeProblemAreas]
+            List of objects representing short edge problem areas.
+        """
+        if not bodies:
+            return []
+
+        length_value = DoubleValue(value=float(length))
+        body_ids = [body.id for body in bodies]
+        problem_areas_response = self._repair_stub.FindShortEdges(
+            FindShortEdgesRequest(
+                selection=body_ids, max_edge_length=length_value
+            )
+        )
+
+        parent_design = get_design_from_body(bodies[0])
+        return [
+            ShortEdgeProblemAreas(
+                str(res.id),
```

Comment:
This should already be a string AFAIK

Reply:
ShortEdgeProblemArea in models.proto transmits id as int32 (as seen [here.](https://github.com/ansys/ansys-api-geometry/blob/main/ansys/api/geometry/v0/models.proto#L268))